mandatory/2-function-creation.js

```diff
@@ -15,27 +23,64 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice >= 200) {
+    return totalPrice * 0.9;
+  }
+  else {
+    return totalPrice * 0.95;
 
-/*
+  }
+
+
+
+}
```

Comment:
Read the requirements carefully, should you be checking if the totalPrice is greater or equal to 200?

Also, the else block should not be on a new line